src/main/scala/xiangshan/backend/fu/FuncUnit.scala

```diff
@@ -215,8 +215,8 @@ trait HasPipelineReg { this: FuncUnit =>
     val validVec = valid +: Seq.fill(latency)(RegInit(false.B))
     val ctrlVec = init.ctrl +: Seq.fill(latency)(Reg(chiselTypeOf(io.in.bits.ctrl)))
     val dataVec = init.data +: Seq.fill(latency)(Reg(chiselTypeOf(io.in.bits.data)))
-    val perfVec = init.perfDebugInfo +: Seq.fill(latency)(Reg(chiselTypeOf(io.in.bits.perfDebugInfo)))
-    val seqNumVec = init.debug_seqNum +: Seq.fill(latency)(Reg(chiselTypeOf(io.in.bits.debug_seqNum)))
+    val perfVec = init.perfDebugInfo.getOrElse(0.U.asTypeOf(new PerfDebugInfo)) +: Seq.fill(latency)(Reg(chiselTypeOf(io.in.bits.perfDebugInfo.getOrElse(0.U.asTypeOf(new PerfDebugInfo)))))
```

Comment:
Replacing +: with ++: is enough to handle this.
Option can be appended in Seq just by using ++: operator or prependedAll function.